xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -1166,6 +1166,11 @@ def _write_spec(
             continue
         target_hosts.setdefault(s.hosted, []).append(s.host)
 
+    alias_labels = {
```

Comment:
@brentleyjones we discussed using this in other places e.g. `owned_extra_files`, I can push that next but curious to know if you know of other places we could expand? Or a method to find what else needs to be expanded maybe?

Reply:
I think it might only be `owned_extra_files`.